en/proton.html

```diff
@@ -31,7 +31,7 @@ <h3 id="bucket-management">Bucket management</h3>
 As the distributors wants quick access to bucket metadata,
 it keeps an in-memory bucket database to efficiently serve these requests.
 </p><p>
-It implements elasticity support in terms of the SPI.
+It implements elasticity support in terms of the <a href="jdisc/pluggable-frameworks.html">SPI</a>.
```

Comment:
The SPI in that link and the SPI discussed in this document are actually two separate entities. Although conceptually related (from a software design perspective), the Proton SPI is an internal implementation detail that lives in C++ land, whereas the jDISC SPI page relates to (user-) pluggable Java components.

It is fully understandable why this is not obvious, since this documentation page does not even bother to mention what "SPI" means in its context 🙈 There's quite a bit of outdated information on this page. I already have a set of documentation pages that I intend to update this summer—I can add this to the list. 🏝️

Reply:
Thanks for clarifying! Let me know if I can help with anything on this front.

I'm guessing we can close this PR in the meantime?

Reply:
> Let me know if I can help with anything on this front.

Will do!

> I'm guessing we can close this PR in the meantime?

Yes, this documentation page needs a round with the high pressure washer first 🙂